tests/baselines/reference/paramTagNestedWithoutTopLevelObject.errors.txt

```diff
@@ -0,0 +1,12 @@
+tests/cases/conformance/jsdoc/paramTagNestedWithoutTopLevelObject.js(2,20): error TS8032: Qualified name 'xyz.p' is not allowed without a leading '@param {object} xyz'.
+
+
+==== tests/cases/conformance/jsdoc/paramTagNestedWithoutTopLevelObject.js (1 errors) ====
+    /**
+     * @param {number} xyz.p

```

Comment:
can you also add test case for `xyz.bar.p` when there is
 -  `xyz.bar` but no `xyz` 
 -  `xyz` but no `xyz.bar`
- no `xyz` or `xyz.bar`